extension/lib/webcompat.js

```diff
@@ -85,20 +86,45 @@ class WebCompat {
   /**
    * @param {String} elementName -
    *                e.g. div, main, aside
+   * @param {Array} attributes -
+   *                e.g. [{ name: "dir", value: "ltr" }, ...]
    * @param {Array} browsers -
    *                e.g. [{ id: "firefox", name: "Firefox", version: "68" }, ...]
-   * @return {Object} if no issue found, return null.
+   * @return {Array} issues
```

Comment:
Maybe expand this comment to also say that if there are no issues found, then the array is empty.

Reply:
Yep, thanks!